2020/10/2020-10-24-Fuck-the-RIAA.md

```diff
@@ -0,0 +1,102 @@
+October 23, 2020
+
+GitHub
+
+Dear Sir or Madam:
+
+I am contacting you on behalf of the Free Software community and its members.
+
+Fuck the M-P-double-A comin' straight out the underground
+A young pirate got it bad cause I'm down,
+Loadin' DVDs like a motherfuckin' fiend,
+Bring my camera to the movies and I put em on BT,
+Back all that stolen content up on DVD-ROMs,
+'Cuz my tip's been piracy since I dropped out my Mom's,
+And just because I share my MP3s,
+They got the government comin' after me,
+Instead of suin' kids why don't you step on up,
+And release a couple albums that don't completely suck,
+Stop puttin DRM onto audio CDs,
+That don't make it any harder to steal your MP3s,
```

Comment:
Isn't MP3 a proprietary format eye

Reply:
MP3 is no longer a proprietary format, all patents expired in 2016

https://www.theregister.com/2017/05/16/mp3_dies_nobody_noticed/